src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -4122,7 +4122,23 @@ internal static IEnumerable<CompletionResult> CompleteFilename(CompletionContext
                     // can succeed.  This call will mount the drive if it wasn't already.
                     executionContext.SessionState.Drive.GetAtScope(wordToComplete.Substring(0, 1), "global");
                 }
+#if UNIX
+                StringBuilder sb = new StringBuilder();
```

Comment:
You can reasonably assume all characters are letters and pass `wordToComplete.Length * 4` when constructing the string builder to avoid growing and copying the buffer.

Reply:
Thabks! Fixed.